.htaccess

```diff
@@ -39,4 +39,4 @@ RewriteRule ^courses/([^/]+)/course-pic.png$ app/courses/$1/course-pic.png [QSA,
 RewriteRule ^session/(\d{1,})/about/?$ main/session/about.php?session_id=$1 [L]
 
 # Issued badges friendly URL
-RewriteRule ^badge/(\d{1,})/user/(\d{1,}) main/badge/issued.php?skill=$1&user=$2 [L]
+RewriteRule ^badge/issue/(\d{1,})/user/(\d{1,})/? main/badge/issued.php?issue=$1&user=$2 [L]
```

Comment:
You cannot change previous rules like that. You have to maintain previous redirects just in case someone is still pointing at the old URL, and mention that this is the "old" version.